docs/usage/configuration-options.md

```diff
@@ -1810,6 +1810,10 @@ Add to this object if you wish to define rules that apply only to patch updates.
 
 Add to this object if you wish to define rules that apply only to PRs that pin dependencies.
 
+## pinDigest
+
+If enabled renovate will pin the digest value.
+
 ## pinDigests
```

Comment:
why does pinDigestes appear twice? line 1813 and 1817?

Reply:
@viceice is the `pinDigests` type should be the type used in this case ? or is it right to create that new type

Reply:
🤔 We should probably unify those. @rarkins WDYT?

Reply:
I don't see any need for this new option. `pinDigest` is an updateType, while `pinDigests` is the configuration option.

Reply:
@rarkins why do we have updateType and configurationType in the same class?, and both are available for customers in the docs?
is there another way to use the UpdateType if not here?

Reply:
I don't see any hits for `configurationType` in our source code so don't know what you're referring to

Reply:
Any updates here ? Are we leaving these two config separate ?

Reply:
`pinDigests` is a configuration option to say "I want to pin digests". Whereas the option `pinDigest` - if used, is a shortcut for configuration to apply for such updates.

e.g. someone's config might looks like:

```json
{
  "pinDigests": true,
  "pinDigests": {
    "labels": ["digest pinning"]
  }
}
```